config/config.go

```diff
@@ -0,0 +1,228 @@
+// Copyright © 2022 Meroxa, Inc. & Yalantis
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package config defines configurable values shared between Source and Destination
+// and implements a method to parse them.
+package config
+
+import (
+	"fmt"
+	"net/url"
+	"strings"
+	"time"
+
+	"github.com/conduitio-labs/conduit-connector-mongo/validator"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+var (
```

Comment:
The `defaultConnectionURI` cannot be a constant because it's a `*url.URL`. But as for the `defaultServerSelectionTimeout` - sure, I've made it a constant.